src/objects.rs

```diff
@@ -340,6 +345,32 @@ impl Id<StopArea> for StopPoint {
 }
 impl_codes!(StopPoint);
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FeedInfo {
+    #[serde(rename = "feed_info_param")] pub info_param: String,
+    #[serde(rename = "feed_info_value")] pub info_value: String,
+}
```

Comment:
No pub as it's only used to decode the CSV. Thus please put it private in ntfs.rs.

Reply:
Are you sure? I need to read the properties in ntfs.rs

Reply:
If you move it to ntfs.rs, pub is not needed, yes.